modern/base/Dockerfile

```diff
@@ -96,6 +98,7 @@ RUN apt-get -qq update \
     && rm -rf /tmp/* \
     && runuser -u conan -- printf 'eval "$(pyenv init -)"\n' >> /home/conan/.bashrc \
     && runuser -u conan -- printf 'eval "$(pyenv virtualenv-init -)"\n' >> ~/.bashrc \
```

Comment:
btw, where is this line printed? not sure if `~` resolves to `/home/conan` here (cc/ @uilianries)

Reply:
Thank you, you found a bug. It was sent to /root/.bashrc